site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -2493,6 +2493,38 @@ public function ajaxSetPeerFeedback($gradeable_id) {
         }
         $gradeable->setPeerFeedback($grader_id, $user_id, $feedback);
         $this->core->getOutput()->renderJsonSuccess("Feedback successfully uploaded");
```

Comment:
It looks like this used to return true before you fixed your merge conflicts.

Reply:
Readded the return true